Bottom: 4d2a9ae39c62a8c793af91f810421b06e4868822
Top:    f334ecbcee46778e1a438b2bdf1ed15d21b9451f
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-05 16:51:02 +0800

Refresh of virtio-net-rename-virtnet_xdp

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 1cebf61..4507714 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -2362,9 +2362,9 @@ static u32 virtnet_xdp_query(struct net_device *dev)
 	return 0;
 }
 
-static int virtnet_xdp(struct net_device *dev, struct netdev_bpf *xdp)
+static int virtnet_bpf(struct net_device *dev, struct netdev_bpf *bpf)
 {
-	switch (xdp->command) {
+	switch (bpf->command) {
 	case XDP_SETUP_PROG:
 		return virtnet_xdp_set(dev, xdp->prog, xdp->extack);
 	case XDP_QUERY_PROG:
@@ -2454,7 +2454,7 @@ static const struct net_device_ops virtnet_netdev = {
 #ifdef CONFIG_NET_POLL_CONTROLLER
 	.ndo_poll_controller = virtnet_netpoll,
 #endif
-	.ndo_bpf		= virtnet_xdp,
+	.ndo_bpf		= virtnet_bpf,
 	.ndo_xdp_xmit		= virtnet_xdp_xmit,
 	.ndo_features_check	= passthru_features_check,
 	.ndo_get_phys_port_name	= virtnet_get_phys_port_name,
